### PR TITLE
feat: Option to use lua 5.4.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

**Review by aaronkvanmeerten — APPROVED** [review]

File: ansible/roles/prosody/tasks/install.yml [hunk 1]
```diff
@@ -64,6 +58,44 @@
     - inspect
     - uuid
 
+
+- name: Install lua 5.5
```
Comment (aaronkvanmeerten) [inline ansible/roles/prosody/tasks/install.yml:62]:
```suggestion
- name: Install lua 5.4
```

File: ansible/roles/prosody/tasks/install.yml [hunk 2]
```diff
@@ -64,6 +58,44 @@
     - inspect
     - uuid
 
+
+- name: Install lua 5.4
+  ansible.builtin.apt:
+    name: ["lua5.4", "liblua5.4-dev"]
+    state: present
+
+- name: Set default lua 5.4 and luarocks to use 5.4
```
Comment (bgrozev) [inline ansible/roles/prosody/tasks/install.yml:67]: Does this also need a `when: prosody_use_lua_54` like the one in configure.yml?

**Review by bgrozev — APPROVED** [review]

File: ansible/roles/prosody/tasks/install.yml (hunk 2 above)
Comment (bgrozev) [reply ansible/roles/prosody/tasks/install.yml:67]: Looks like no :)

Comment (damencho) [reply ansible/roles/prosody/tasks/install.yml:67]: Nope as this is install time, we want during configure to have the option to chose, so on install we install both.

**Comment by damencho** [comment]
I'm in the process of testing those before merging.

**Pull request merged**